roles/loki/tasks/main.yml

```diff
@@ -106,7 +106,7 @@
     grafana_user: admin
     grafana_password: "{{ grafana_password }}"
     ds_type: loki
-    ds_url: http://localhost:3100
+    ds_url: http://172.16.34.1:3100
```

Comment:
As a rule of thumb, every time you notice yourself changing a particular part in a role, that's where it makes sense to make it a variable. Our future selves will thank us!